docs/src/index.md

```diff
@@ -5,16 +5,100 @@ matrices that have uncertain parameters.
 
 ## Features
 
-- An `IntervalMatrix` type.
-- Arithmetics between scalars, closed intervals and interval matrices.
-- Overapproximation and underapproximation routines for the exponential of an
-  interval matrix at different orders.
+Here is a quick summary of the available functionality.
+See the section `Library Outline` below for details.
 
-## Library Outline
+- An `IntervalMatrix` type that wraps a two-dimensional array whose components
+  are intervals.
+- Arithmetic between scalars, intervals and interval matrices.
+- Quadratic expansions using single-use expressions (SUE).
+- Interval matrix exponential: underapproximation and overapproximation routines.
+- Utility functions such as: operator norm, random sampling, splitting and
+  containment check.
+
+## Quickstart
+
+An *interval matrix* is a matrix whose coefficients are intervals. For instance,
+
+```julia
+julia> A = IntervalMatrix([0..1 1..2; 2..3 -4.. -2])
+2×2 IntervalMatrix{Float64,Interval{Float64},Array{Interval{Float64},2}}:
+ [0, 1]    [1, 2]
+ [2, 3]  [-4, -2]
+```
+defines an interval matrix $A$. The type of each coefficient in $A$ is an interval,
+e.g. its coefficient in position $(1, 1)$ is the interval $[0, 1]$ over double-precision
+floating point numbers:
+
+```julia
+julia> A[1, 1]
+[0, 1]
+
+julia> typeof(A[1, 1])
+Interval{Float64}
+```
+This library uses the interval arithmetic package
```

Comment:
```suggestion
This library uses the interval-arithmetic package
```

Reply:
i'm not used to seeing a hyphen in interval arithmetic

Reply:
This has nothing to do with "interval arithmetic." [The rule](https://www.grammar-monster.com/lessons/hyphens_in_compound_adjectives.htm) is: if you connect three or more words, you put a hyphen if the words should bind to the left.